src/Libraries/Microsoft.Extensions.AI.Abstractions/ChatCompletion/ChatResponse.cs

```diff
@@ -3,58 +3,90 @@
-    /// <param name="message">The chat message representing the singular choice in the response.</param>
+    /// <param name="message">The response message.</param>
+    /// <exception cref="ArgumentNullException"><paramref name="message"/> is <see langword="null"/>.</exception>
     public ChatResponse(ChatMessage message)
     {
         _ = Throw.IfNull(message);
-        _choices = [message];
+        Messages.Add(message);
     }
 
-    /// <summary>Gets or sets the list of chat response choices.</summary>
-    public IList<ChatMessage> Choices
+    /// <summary>Initializes a new instance of the <see cref="ChatResponse"/> class.</summary>
+    /// <param name="messages">The response messages.</param>
+    /// <exception cref="ArgumentNullException"><paramref name="messages"/> is <see langword="null"/>.</exception>
+    /// <exception cref="ArgumentException"><paramref name="messages"/> must not be read-only.</exception>
+    public ChatResponse(IList<ChatMessage> messages)
+    {
+        _ = Throw.IfNull(messages);
+        _ = Throw.IfReadOnly(messages);
+
+        _messages = messages;
+    }
+
+    /// <summary>Gets or sets the chat response messages.</summary>
+    /// <exception cref="ArgumentException">The <see cref="IList{T}"/> must not be read-only.</exception>
+    public IList<ChatMessage> Messages
     {
-        get => _choices;
-        set => _choices = Throw.IfNull(value);
+        get => _messages ??= new List<ChatMessage>(1);
+        set
+        {
+            if (value is not null)
+            {
+                _ = Throw.IfReadOnly(value);
+            }
+
+            _messages = value;
+        }
     }
 
-    /// <summary>Gets the chat response message.</summary>
+    /// <summary>Gets the text of the response.</summary>
     /// <remarks>
-    /// If there are multiple choices, this property returns the first choice.
-    /// If <see cref="Choices"/> is empty, this property will throw. Use <see cref="Choices"/> to access all choices directly.
+    /// This property concatenates the <see cref="ChatMessage.Text"/> of all <see cref="ChatMessage"/>
+    /// instances in <see cref="Messages"/>.
     /// </remarks>
     [JsonIgnore]
-    public ChatMessage Message
+    public string Text
     {
         get
         {
-            var choices = Choices;
-            if (choices.Count == 0)
+            IList<ChatMessage>? messages = _messages;
+            if (messages is null)
             {
-                throw new InvalidOperationException($"The {nameof(ChatResponse)} instance does not contain any {nameof(ChatMessage)} choices.");
+                return string.Empty;
             }
 
-            return choices[0];
+            int count = messages.Count;
+            return count switch
+            {
+                0 => string.Empty,
+                1 => messages[0].Text,
+                _ => messages.SelectMany(m => m.Contents).ConcatText(),
```

Comment:
Do you think there would be a case for having at least a linebreak between the text from successive messages?

It looks like, as-is, separate messages would be combined without any spacing at all, so a message sequence like:

 - I'll see if I can find information for you.
 - Sorry, there was no information.

... would appear as: `I'll see if I can find information for you.Sorry, there was no information.` with incorrectly missing spacing between the two sentences.

I know this pulls us into taking an opinion about text formatting which we might prefer to avoid, but without having *some* kind of spacing it may just seem buggy. A linebreak might be the most culture-agnostic way to separate two blocks of text.